worlds/stardew_valley/content/mods/archeology.py

```diff
@@ -1,10 +1,19 @@
 from ..game_content import ContentPack
 from ..mod_registry import register_mod_content_pack
+from ...data.game_item import ItemTag, Tag
+from ...data.shop import ShopSource
 from ...data.skill import Skill
 from ...mods.mod_data import ModNames
+from ...strings.book_names import ModBook
+from ...strings.region_names import LogicRegion
 from ...strings.skill_names import ModSkill
 
 register_mod_content_pack(ContentPack(
     ModNames.archaeology,
-    skills=(Skill(name=ModSkill.archaeology, has_mastery=False),)
+    skills=(Skill(name=ModSkill.archaeology, has_mastery=False),),
```

Comment:
No mastery for the archaeology skill? Just want to make sure it was not forgotten. I don't think I've changed the logic rules yet to use the skills from the content packs, but eventually this will drive the amount of progression skill items.

Reply:
I don't think any skill mods have created masteries yet

Reply:
Its planned but not yet implemented.